docs/content/developers/building-contributing.md

```diff
@@ -19,6 +19,10 @@ Each [PR build](https://github.com/ddev/ddev/actions/workflows/pr-build.yml) cre
 
 Download and unzip the appropriate binary and place it in your `$PATH`.
 
+## Install a previous version
+
+You can also [install an old version of DDEV](https://ddev.readthedocs.io/en/stable/users/usage/faq/#how-can-i-install-a-specific-version-of-ddev).
```

Comment:
Thank you, I propose to add two more word for search: rollback and downgrade.

```suggestion
## Rollback to a previous version

You can also [downgrade to a previous version of DDEV](../users/usage/faq.md#how-can-i-install-a-specific-version-of-ddev).
```

Reply:
Great suggestion, thanks @stasadev!